mungegithub/github/github.go

```diff
@@ -726,40 +747,45 @@ func (obj *MungeObject) RemoveLabel(label string) error {
 // GetHeadAndBase returns the head SHA and the base ref, so that you can get
 // the base's sha in a second step. Purpose: if head and base SHA are the same
 // across two merge attempts, we don't need to rerun tests.
-func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool) {
+//CHECK-UPDATED
+func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool, err error) {
```

Comment:
@lavalamp what do you think of doing more `ok bool` returns from `github/github.go` fucntions instead of `error` returns? Should we move towards that style? We seem to very very rarely ever care what the error is in the caller...